docs/hw/bus/pmc.rst

```diff
@@ -282,16 +282,26 @@ no effect and currently unknown purpose:
    Has the same bits as ENABLE, comes up as all-1 on boot, except for PDISPLAY
    bit which comes up as 0.
 
-.. reg:: 32 pmc-enable-unk0c ??? related to enable
-
-   Has bits which correspond to PFIFO engines in ENABLE, ie.
-
-   - 1: PPPP
-   - 6: PCOPY[0]
-   - 7: PCOPY[1]
-   - 12: PGRAPH
-   - 15: PVLD
-   - 17: PPDEC
+.. reg:: 32 pmc-elpg-enable ELPG enables
+
+   Enable engine-level power gating (ELPG) controllers. Bits correspond to PFIFO
+   engines in ENABLE, ie.
+
+   - 1: PPPP [GP100-]
+   - 2: PXBAR [GK104-]
+   - 3: PMFB  [GP100-]
+   - 6: PCOPY[0] [GP100-]
+   - 7: PCOPY[1] [GP100-]
+   - 12: PGRAPH [GP100-]
```

Comment:
Uh.

PGRAPH (and the other bits that were there before) are present since GF104 as well.

Not sure about the other bits...